### PR TITLE
Add shoovify.sh to shoov.io

### PR DESCRIPTION
**Pull request by solker**
https://github.com/shoov/shoov/issues/34

Public:

`curl -s http://shoov.io/shoovly.sh | sh`

Dev:

`curl -s http://localhost:3000/shoovly.sh | sh`

Gist:
https://gist.github.com/solker/3999cd675b16c47329a5

### CODE REVIEW

**Comment by solker** [comment]
@amitaibu 
- Realized condition for existing behat folder.
- Realized commit process. 

To test:
`curl -sL https://goo.gl/SykDlu | sh`

Bug: 
Read construction not working while file running after curl. Need change it to something else or figure out why.

**Comment by amitaibu** [comment]
So for now leave the `read` out. Also lets prettify the echo a bit.

**Comment by solker** [comment]
@amitaibu
Another approach work with prompt:
`bash <(curl -sL https://goo.gl/SykDlu)`

**Comment by solker** [comment]
@amitaibu 
or something like that
`curl -s http://shoov.io/shoovly.sh > shoov.sh && sh shoov.sh && rm -r shoov.sh`

selection_085

**Comment by amitaibu** [comment]
No, lets stay with the usual pattern for now.

**Comment by solker** [comment]
@amitaibu ready for code review and merge.

Clean installation (without behat directory)
selection_086
selection_087

Not clean installation (with behat directory)
selection_089

File: src/shoovify.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+git status
+
+# commit
+git commit -am 'Shoovify complete!'
+
+echo
+echo "${YELLOW}Shoov is configured. Now you need to make push and after";
+echo "you can activate your repository on the shoov.io${END}";
+echo
+echo "Also you can run tests by next command: cd behat && bin/behat"
```
Comment (amitaibu) [inline src/shoovify.sh:66]: Make this yellow as-well

**Comment by amitaibu** [comment]
Merged.

**Comment by amitaibu** [comment]
@niryariv you can now `curl -s http://shoov.io/shoovify.sh | sh` from the root of your _existing_ repo

**Pull request closed without merging**